src/components/elements/buttons/SortUpButton.js

```diff
@@ -0,0 +1,12 @@
+import classes from "./SortButton.module.css";
```

Comment:
Feels a bit over-engineered to split Sort Up/Down at this point when it is just a basic button.

Reply:
Was thinking because I'll have many different sorts (Date, author, likes, content length etc.) it would be useful